pages/common/openssl-genpkey.md

```diff
@@ -0,0 +1,16 @@
+# openssl genpkey
+
+> OpenSSL command to generate asymmetric key pairs.
+> More information: <https://www.openssl.org/docs/man1.0.2/man1/genpkey.html>.
+
+- Generate a 2048bit RSA private key and save it to a file:
+
+`openssl genpkey -algorithm rsa -pkeyopt rsa_keygen_bits:2048 -out {{filename.key}}`
+
+- Generate an elliptic curve private key using the curve *prime256v1* and save it to a file:
+
+`openssl genpkey -algorithm EC -pkeyopt ec_paramgen_curve:prime256v1 -out {{filename.key}}`
```

Comment:
```suggestion
`openssl genpkey -algorithm {{EC}} -pkeyopt {{ec_paramgen_curve}}:{{prime256v1}} -out {{filename.key}}`
```

Reply:
Same comment as for the RSA key generation. The only token here should be `{{prime256v1}}`

Reply:
Perhaps this will read better?
```suggestion
`openssl genpkey -algorithm EC -pkeyopt ec_paramgen_curve:{{prime256v1}} -out {{filename.key}}`
```